app/notifications/process_notifications.py

```diff
@@ -365,6 +365,8 @@ def persist_notifications(notifications: List[VerifiedNotification]) -> List[Not
             )
         )
     bulk_insert_notifications(lofnotifications)
+    # bulk_insert_notifications(lofnotifications)
```

Comment:
Uncommenting this will cause a SQLAlchemy error and log.

Delete this line before merging.